src/components/PostList.tsx

```diff
@@ -26,76 +34,58 @@ const PostList = ({ data, user }: PostListProps): JSX.Element => {
-                <FaDotCircle aria-hidden="true" className="mr-1"/>
-                {data.issues && <p className="font-bold">{humanizeNumber(data.issues)}</p>}
-              </div>
-            </a>
-
+        <div className='flex-1'>
             <a
-              className="flex justify-start  text-xs sm:text-xl text-grey transition-all duration-200 w-16 sm:w-24"
-              href={getRepoLink(data.full_name)}
-              title={`Add a star to ${data.full_name}`}
+              href={getRepoLink(full_name)}
+              title={`Visit ${full_name}`}
               target="_blank"
               rel="noopener"
-            >
-              <div className="cursor-pointer flex justify-start items-center hover:text-saucyRed transition-all duration-200">
-                <FaStar aria-hidden="true" className="mr-1"/>
-                {data.stars && <p className="font-bold">{humanizeNumber(data.stars)}</p>}
-              </div>
+              >
+                <p className='text-[14px] text-textGrey'>{full_name}</p>
+                <p className='text-[16px] text-textGrey'>{description}</p>
             </a>
-          </div>
+            <div className='flex gap-x-[16px] mt-[16px]'>
+                <div className='flex gap-[5px] items-center text-textGrey'>
+                    <img className='w-[16px]' src={issueIconGrey} alt="issues"/>
+                    <p className='text-[14px]'>{humanizeNumber(issues)}</p>
+                </div>
+                <div className='flex gap-[5px] items-center text-textGrey'>
+                    <img className='w-[16px]' src={starIconGrey} alt="stars"/>
+                    <p className='text-[14px]'>{humanizeNumber(stars)}</p>
+                </div>
+                <div className='flex gap-x-[5px]'>
+                    {
+                      contributions.slice(0, 5).map(({contributor, last_merged_at}) => (
+                        <div className='w-[24px] h-[24px] overflow-hidden rounded-full -mr-[15px] transition-all duration-300 hover:mr-0 hover:scale-105'>
+                          <Avatar contributor={contributor} lastPr={last_merged_at} />
+                        </div>
+                      ))
+                    }
+                </div>
+            </div>
         </div>
-      </div>
+        <button
+        onClick={() => (user_id ? handleVoteUpdateByRepo(votes, repo_id) : signIn({ provider: "github" }))}
+        className='md:w-[60px] w-full min-w-[60px] rounded-[6px] group border-[1px] cursor-pointer transition-all duration-200 hover:border-osOrange flex gap-[5px] py-[10px] md:py-0 md:flex-col justify-center items-center'>
+          <FaArrowAltCircleUp className='text-gray-500 group-hover:text-osOrange transition-all duration-300 w-[13px] h-[13px]'/>
+          <p className='text-[12px] font-semibold text-gray-500 group-hover:text-osOrange transition-all duration-500'>{humanizeNumber(votes)}</p>
```

Comment:
you cannot put a paragraph inside a button, please replace this with any inline item 🍕